config.yml

```diff
@@ -220,7 +220,7 @@ near:
   api: https://staging-rpc.nearprotocol.com
 
 elrond:
-  api: https://api.elrond.com
+  api: https://gateway.elrond.com
```

Comment:
is it only block related only? there is also transactions list and status endpoint are used?

Reply:
Is not a problem if the transactions list is returned by this endpoint. This endpoint is similar with https://api.elrond.com but have some extra API's.